src/components/home/modals/NewToken.vue

```diff
@@ -485,14 +484,14 @@ export default {
                 this.fee = BigNumber(TOKEN_FEE * VSYS_PRECISION)
                 this.feeScale = 100
                 const dataInfo = {
-                    contract: CONTRACT,
+                    contract: this.support === true ? CONTRACT : CONTRACT_WITH_SPLIT,
```

Comment:
1. this.support is always *false* whether supporting split checkbox is checked or not.
2. if this.support is true, contract should use CONTRACT_WITH_SPLIT, not CONTRACT